src/components/MapImageSlider.vue

```diff
@@ -1813,128 +1813,123 @@
     </caption>
   </div>
 </template>
-<script>
+<script setup>
+  import { onMounted } from "vue";
+
   import BeerSlider from "beerslider";
-  import GeorgiaInsetMap from './georgiaInsetMap';
-  import atlantaSliderText from "../assets/mapboxSlider/atlantaSliderText";
-  export default {
-      'name': 'MapImageSlider',
-      components:{
-            GeorgiaInsetMap
-      },
-      data(){
-          return{
-              atlantaText: atlantaSliderText.textContents,
-              svg: null,
-              pt: null,
-              firstHover: null
-          }
-      },
-      mounted(){
-          window.addEventListener('load', function(){
-              new BeerSlider(document.getElementById('sliderOne'));
-              new BeerSlider(document.getElementById('sliderTwo'));
-          })
-          this.svg = document.querySelector("#cartogram-svg");
-          this.pt = this.svg.createSVGPoint();
-      },
-      methods: {
-      
-        cursorPoint(evt) {
-          this.pt.x = evt.clientX; this.pt.y = evt.clientY;
-          return this.pt.matrixTransform(this.svg.getScreenCTM().inverse());
-        },
+  import GeorgiaInsetMap from '@/components/GeorgiaInsetMap.vue';
```

Comment:
```suggestion
  import GeorgiaInsetMap from '@/components/georgiaInsetMap.vue';
```

Reply:
needed to get page to build

Reply:
weird - I changed the capitalization in the filename locally, but apparently that didn't update in git

Reply:
ok I made this edit in GitHub - should now work as is, retaining the component naming convention of starting w/ a capitalized letter